src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed') || false;
```

Comment:
@whipermr5 Just as a continuation to the [point I made above](https://github.com/TEAMMATES/teammates/pull/7811#discussion_r131247789), I think the variable name makes it clear enough about which side is receiving the benefit of the doubt either way.